src/components/NumberIcon/NumberIcon.tsx

```diff
@@ -48,7 +48,7 @@ export interface Props {
 export const NumberIcon = ({
   className,
   number,
-  size = 'lg',
```

Comment:
Now that there's no default `size`, line 65 is adding the style `number-icon--undefined` in your snapshots.